### PR TITLE
EZP-24706: Replace content loading by contentinfo

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-24706

## Description
ContentInfo is now available in location. The goal of this PR is to remove content loading everywhere it is no longer mandatory in order to provide better performances.

## Test
Manual test and update of existing unit tests

## TODO
- [x] Content structure and media library links
- [x] Breadcrumbs
- [x] Content tree
- [x] Figure out why Content tree didn't break any tests -> because it is not covered :(
- [x] Cleanup and rebase

### CODE REVIEW

**Comment by andrerom** [comment]
Cool :)

**Comment by bdunogier** [comment]
Ewww, it's full of js in here, you should add a big warning sign, people might get hurt.

Good luck with that :)

**Comment by yannickroger** [comment]
Ready for review ping @dpobel @mhyndle @StephaneDiot

File: Resources/public/js/views/ez-locationviewview.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ YUI.add('ez-locationviewview', function (Y) {
             var title = this.get('content').get('name');
 
             return Y.Array.reduce(this.get('path'), title, function (title, val) {
-                return title + ' / ' + val.content.get('name');
+                return title + ' / ' + val.get('contentInfo').get('name');
```
Comment (dpobel) [inline Resources/public/js/views/ez-locationviewview.js]: you also have to update the `path` attribute documentation

File: Resources/public/js/views/services/ez-navigationhubviewservice.js [hunk 2]
```diff
@@ -155,17 +150,7 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
...
-
-                content.load(loadOptions, function (error) {
-                    if ( error ) {
-                        callback(error);
-                        return;
-                    }
-
-                    callback();
-                });
+                callback();
```
Comment (dpobel) [inline Resources/public/js/views/services/ez-navigationhubviewservice.js]: you can write directly write something like:
```js
location.load(loadOptions, callback);
```

**Comment by yannickroger** [comment]
@dpobel 's feedback taken into account

**Comment by dpobel** [comment]
+1

**Comment by mhyndle** [comment]
+1

**Pull request merged**